schema.graphql

```diff
@@ -493,3 +493,14 @@ type NftTxCount @entity {
 
   count: BigInt!
 }
+
+# ID: `${contractAddress}-${ownerAddress}-${operatorAddress}`
```

Comment:
ID shouldn't include operator here, as `${contractAddress}-${ownerAddress}` is enough to identify.  If we want to find the approved operator, we should only need `${contractAddress}-${ownerAddress}`

Reply:
look at it like: 
key = `${contractAddress}-${ownerAddress}`
value = `${operatorAddress}`

Reply:
Just updated. Let me know if you see anything else.

Reply:
@robsecord This change is made so let me know if there's anything else on this one or if it can be merged.